voting/templates/base.html

```diff
@@ -73,5 +96,21 @@
 <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"
         integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6"
         crossorigin="anonymous"></script>
+
+<script>
+    function openSlideMenu() {
```

Comment:
Put these in a separate js file under the static folder (where style.css is located) and reference it with <script ...>